src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/PartialViewResultExecutor.cs

```diff
@@ -158,5 +160,47 @@ public virtual Task ExecuteAsync(ActionContext actionContext, IView view, Partia
                 viewResult.ContentType,
                 viewResult.StatusCode);
         }
+
+        private static string GetActionName(ActionContext context)
```

Comment:
That argument could also be applied to controller and area, what's special about ActionName? I think it's more correct that the view engine looks for default views based on data from routing. The aim with this change is to stop treating ActionName specially.